plugins/datetime/Makefile.am

```diff
@@ -5,17 +5,19 @@ polkitdir = $(datadir)/polkit-1/actions
 dbus_services_in_files = org.mate.SettingsDaemon.DateTimeMechanism.service.in
 polkit_in_files = org.mate.settingsdaemon.datetimemechanism.policy.in
 
-msd-datetime-mechanism-glue.h: $(srcdir)/msd-datetime-mechanism.xml
-	$(AM_V_GEN) dbus-binding-tool \
-			--prefix=msd_datetime_mechanism --mode=glib-server	\
-			--output=msd-datetime-mechanism-glue.h			\
-			$(srcdir)/msd-datetime-mechanism.xml
+$(BUILT_SOURCES) : Makefile.am $(srcdir)/msd-datetime-mechanism.xml
```

Comment:
You can remove Makefile.am from prerequisite list, then you could use `$<` instead of `$(srcdir)/msd-datetime-mechanism.xml`:
```
$(BUILT_SOURCES): $(srcdir)/msd-datetime-mechanism.xml
        $(AM_V_GEN) gdbus-codegen \
                --interface-prefix org.mate.SettingsDaemon. \
                --c-namespace MateSettings \
                --generate-c-code msd-datetime-generated $<
```